libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/AppLog.java

```diff
@@ -88,6 +92,22 @@ public static void removeListeners() {
         void onLog(T tag, LogLevel logLevel, String message);
     }
 
+    /**
+     * Add a LogFileWriter that will persist logs to disk
+     * @param context The current application context
+     * @param maxLogCount The maximum number of logs that should be stored
+     */
+     public static void enableLogFilePersistence(Context context, int maxLogCount) {
+         LogFileProvider logFileProvider = LogFileProvider.fromContext(context);
+         new LogFileCleaner(logFileProvider, maxLogCount).clean();
+
+         mLogFileWriter = new LogFileWriter(logFileProvider);
+         mLogFileWriter.write(getAppInfoHeaderText(context) + "\n");
+         mLogFileWriter.write(getDeviceInfoHeaderText(context) + "\n");
+    }
+
+    private static LogFileWriter mLogFileWriter;
```

Comment:
Missed this in the initial review, but Hungarian notation for static properties starts with an `s` for static instead of `m` for member.

```suggestion
    private static LogFileWriter sLogFileWriter;
```